src/main/java/org/waarp/common/crypto/ssl/WaarpSecureKeyStore.java

```diff
@@ -161,6 +168,7 @@ public void initKeyStore(String _keyStoreFilename, String _keyStorePasswd, Strin
         try {
             inputStream = new FileInputStream(keyStoreFilename);
             keyStore.load(inputStream, getKeyStorePassword());
+            validateKeyStore(keyStore);
```

Comment:
Return value should not be discarded: if the server starts with an invalid certificate, it should not start at all.
Here, it will still be used, allowing insecure connections to be possible.